external_licenses/Microsoft.Data.SqlClient.SNI.5.2.0.License.txt

```diff
@@ -0,0 +1,46 @@
+MICROSOFT SOFTWARE LICENSE TERMS
```

Comment:
Please update PR to be a bit more descriptive of what changes it contains. The title gets added to the Release Notes which are difficult to read and understand if it doesn't mention what the changes are.